provider/cmd/leaseStatus.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	ccontext "github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/spf13/cobra"
+
+	"github.com/ovrclk/akash/provider/gateway"
+	mcli "github.com/ovrclk/akash/x/market/client/cli"
+	mtypes "github.com/ovrclk/akash/x/market/types"
+	pmodule "github.com/ovrclk/akash/x/provider"
+)
+
+func leaseStatusCmd(codec *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "lease-status",
+		Short: "get provider status",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return doLeaseStatus(codec, cmd)
+		},
+	}
+
+	cmd.Flags().String("owner", "", "bid owner address to filter")
+	cmd.Flags().Uint64("dseq", 0, "Deployment Sequence")
+	cmd.Flags().Uint32("gseq", 0, "Group Sequence")
+	cmd.Flags().Uint32("oseq", 0, "Order Sequence")
+	cmd.Flags().String("provider", "", "Provider id")
+
+	_ = cmd.MarkFlagRequired("owner")
+	_ = cmd.MarkFlagRequired("dseq")
+	_ = cmd.MarkFlagRequired("gseq")
+	_ = cmd.MarkFlagRequired("oseq")
+	_ = cmd.MarkFlagRequired("provider")
```

Comment:
`mtypes.AddBidIDFlags`

(I've been meaning to add a `AddLeaseIDFlags()`, `LeaseIDFromFlags()` for ages but haven't gotten around to it)